infra/resources/src/main/res/values-fr/strings.xml

```diff
@@ -288,26 +324,32 @@
     <string name="dashboard_sync_card_no_modules_button">Modules</string>
     <string name="dashboard_sync_card_offline_message">Veuillez activer la connexion Internet dans les réglages</string>
     <string name="dashboard_sync_card_offline_button">Réglages</string>
+    <string name="dashboard_sync_card_too_many_modules_message">Trop de modules ont été téléchargés.</string>
+    <string name="dashboard_sync_card_relogin_required">Vous devez vous connecter pour synchroniser.</string>
+
     <string name="dashboard_sync_card_last_sync">Dernière synchronisation : %1$s</string>
     <string name="dashboard_sync_card_records_uploaded">Tous enregistrements téléchargés</string>
+    <plurals name="dashboard_sync_card_records_to_upload">
+        <item quantity="one">%1$d enregistrement à envoyer</item>
+        <item quantity="many">%1$d enregistrements à envoyer</item>
+        <item quantity="other">%1$d enregistrements à envoyer</item>
+    </plurals>
     <string name="dashboard_sync_card_try_again_button">Réessayez</string>
 
     <!-- Dashboard - Daily activity card -->
     <string name="dashboard_activity_card_activity">Activité : %1$s</string>
-
-    <!--Dashboard always excplicitly displays the number of activities, thus ignoring the ImpliedQuantity-->
-    <plurals name="dashboard_activity_card_enrolments" tools:ignore="ImpliedQuantity">
-        <item quantity="one">Inscription</item>
+    <plurals name="dashboard_activity_card_enrolments">
+        <item quantity="one">%d Inscription</item>
```

Comment:
We don't have the `%d` modifier in English version. Wouldn't it lead to a runtime crash?
![image](https://github.com/Simprints/Android-Simprints-ID/assets/129499142/d69c1f43-79c5-46ed-b3a1-5278b0733130)

Reply:
I just realised that since we are likely not providing the argument for the placeholder there, it would just print out the "%d" as is.